spec/mainadditions.html

```diff
@@ -212,91 +208,167 @@ <h1>
-          1. Let _finalDate_ be MakeDate(MakeDay(_yr_, _m_, _dt_), MakeTime(_h_, _min_, _s_, _milli_)).
-          1. Let _dv_ be TimeClip(UTC(_finalDate_)).
-        1. Let _O_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%Date.prototype%"*, « [[DateValue]] »).
-        1. Set _O_.[[DateValue]] to _dv_.
-        1. Return _O_.
-      </emu-alg>
-    </emu-clause>
-  </emu-clause>
+  <emu-clause id="sec-date-objects">
+    <h1>Date Objects</h1>
 
-  <emu-clause id="sec-temporal-properties-of-the-date-constructor">
-    <h1><a href="https://tc39.es/ecma262/#sec-properties-of-the-date-constructor">Properties of the Date Constructor</a></h1>
+    <emu-clause id="sec-overview-of-date-objects-and-definitions-of-abstract-operations">
+      <h1>Overview of Date Objects and Definitions of Abstract Operations</h1>
 
-    <emu-clause id="sec-temporal-date.now">
-      <h1><a href="https://tc39.es/ecma262/#sec-date.now">Date.now ( )</a></h1>
-      <del class="block">
-        <p>This function returns the time value designating the UTC date and time of the occurrence of the call to it.</p>
-      </del>
-      <ins class="block">
-        <p>This function performs the following steps when called:</p>
+      <p>[...]</p>
+
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are represented by <dfn variants="time zone identifier">time zone identifiers</dfn>, which are Strings composed entirely of code units in the inclusive interval from 0x0000 to 0x007F.
+          Time zones supported by an ECMAScript implementation may be <dfn variants="available named time zone">available named time zones</dfn>, represented by the [[Identifier]] field of the Time Zone Identifier Records returned by AvailableNamedTimeZoneIdentifiers, or <dfn variants="offset time zone">offset time zones</dfn>, represented by Strings for which IsTimeZoneOffsetString returns *true*.
+          <ins>Time zone identifiers are compared using ASCII-case-insensitive comparisons.</ins>
+        </p>
+        <p>
+          A <dfn variants="primary time zone identifiers">primary time zone identifier</dfn> is the preferred identifier for an available named time zone.
+          A <dfn variants="non-primary time zone identifiers">non-primary time zone identifier</dfn> is an identifier for an available named time zone that is not a primary time zone identifier.
+          An <dfn variants="available named time zone identifiers">available named time zone identifier</dfn> is either a primary time zone identifier or a non-primary time zone identifier.
+          Each available named time zone identifier is associated with exactly one available named time zone.
+          Each available named time zone is associated with exactly one primary time zone identifier and zero or more non-primary time zone identifiers.
+        </p>
+
+        <ins class="block">
```

Comment:
Other than the one line noted above, this `<ins>` block is the only change in this section. The rest is already in 262.